OpenRA.Game/Platform.cs

```diff
@@ -170,11 +170,21 @@ public static void OverrideSupportDir(string path)
 			userSupportPath = path;
 		}
 
+		public static string EngineDir
+		{
+			get
+			{
+				return AppDomain.CurrentDomain.BaseDirectory;
+			}
+		}
+
 		public static string GameDir
 		{
 			get
 			{
-				var dir = AppDomain.CurrentDomain.BaseDirectory;
+				var gameDir = Environment.GetEnvironmentVariable("GAME_DIR");
```

Comment:
It took me a while to find this -- would it make sense to expose an `Engine.GamePath` argument like we do for `Engine.LaunchPath`?